content/en/docs/reference/access-authn-authz/rbac.md

```diff
@@ -279,7 +279,7 @@ rules:
 ```
 
 {{< note >}}
-You cannot restrict `create` or `deletecollection` requests by resourceName. For `create`, this
+You cannot restrict `create` or `deletecollection` or `list` requests by resourceName. For `create`, this
```

Comment:
list and watch actually _can_ be restricted by resourceName, and permits "lists" requests using a fieldSelector to "list" a single object (for example, `/api/v1/namespaces/$ns/configmaps?fieldSelector=metadata.name=foo`)

This is useful when using an informer to list/watch a single object

Reply:
Today I learned!
What might help, then, is to document that if you restrict `list` by `resourceName`, the user / client _must_ query using that field selector (is that right?)

Reply:
yup

Reply:
According to the test I mentioned above, it is possible to use list for a specific resource. But how to use watch?

Reply:
including the `watch` verb in the role would allow watching a specific resource by name using a fieldSelector

Reply:
Forgive me for the mistake.😂 The documentation has been updated